schema/schema.json

```diff
@@ -0,0 +1,348 @@
+                            "description": "zipcode",
+                            "type": "string"
+                        },
+                        "country": {
+                            "description": "country",
+                            "type": "string"
+                        },
+                        "latitude": {
+                            "description": "latitude",
+                            "type": "number"
+                        },
+                        "longitude": {
+                            "description": "longitude",
+                            "type": "number"
+                        },
+                        "sitename": {
+                            "type": "string"
+                        }
+                    }
+                },
+                "group_communities": {
+                    "description": "community",
+                    "type": "array",
+                    "items": {
+                        "type": "string"
+                    }
+                },
+                "role": {
+                    "description": "role of this perfsonar host - regional, science-dmz, etc",
+                    "type": "string",
+                    "enum": [
+                        "nren",
+                        "regional",
+                        "site-border",
+                        "site-internal",
+                        "science-dmz",
+                        "exchange-point",
+                        "test-host",
+                        "default-path",
+                        "backup-path"
+                    ]
+                },
+                "access_policy": {
+                    "description": "access policy for the host. In other words, this defines who can run tests to the host",
+                    "type": "string",
+                    "enum": [
+                        "public",
+                        "private",
+                        "research-education",
+                        "limited"
+                    ]
+                },
+                "access_notes": {
+                    "type": "string"
+                },
+                "pscheduler_service": {
+                    "allOf": [
+                        {
+                            "$ref": "#/$definitions/pscheduler_service_schema"
+                        }
+                    ]
+                },
+                "archive_service": {
+                    "$ref": "#/$definitions/archive_service_schema"
+                },
+                "meta": {
+                    "type": "object"
+                }
+            },
+            "unevaluatedProperties": false,
+            "required": [
+                "administrators",
+                "client_uuid",
+                "access_policy",
+                "pscheduler_service",
+                "name"
+            ]
+        },
+        "service": {
+            "type": "object",
+            "properties": {
+                "name": {
```

Comment:
I don't think we need name anymore. I am not even sure what we would put here for a pscheduler server or an archive.